tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_provision.c

```diff
@@ -172,6 +205,45 @@ static void test_provisioner_pb_adv_multi(void)
 	PASS();
 }
 
+static void test_provisioner_pb_adv_reprovision(void)
+{
+	int err;
+
+	k_sem_init(&prov_sem, 0, 1);
+
+	bt_mesh_device_setup();
+
+	err = bt_mesh_cdb_create(test_net_key);
+	ASSERT_OK(err, "Failed to create CDB (err %d)\n", err);
+
+	err = bt_mesh_provision(test_net_key, 0, 0, 0, 0x0001, dev_key);
+	ASSERT_OK(err, "Provisioning failed (err %d)", err);
+
+	for (int i = 0; i < PROV_REPROV_COUNT; i++) {
+		LOG_INF("Prov'ner prov loop #%d, waiting for prov ...\n", i);
```

Comment:
top of the mornin', Prov'ner :tophat: 
I haven't seen this variant before, I like it :smile:

Reply:
Could be done for the assert message two lines below as well :)